Snowflake.Data.Tests/UnitTests/Tools/UnixOperationsTest.cs

```diff
@@ -126,6 +143,30 @@ public void TestFailIfGroupOrOthersHavePermissionsToFileWithTomlConfigurationVal
             Assert.Throws<SecurityException>(() => s_unixOperations.ReadAllText(filePath, TomlConnectionBuilder.ValidateFilePermissions), "Attempting to read a file with too broad permissions assigned");
         }
 
+        [Test]
+        public void TestFailIfGroupOrOthersHavePermissionsToFileWhileWritingWithUnixValidations([ValueSource(nameof(UserReadWritePermissions))] FilePermissions userPermissions,
```

Comment:
I think you could only make it more visible in the name of the test if it is a test for token cache or toml config.
I would just put something like ForTomlConfig or ForTokenCache in proper tests

Reply:
Done